text2chem/core/formula_parser.py

```diff
@@ -57,7 +57,7 @@ def __get_composition(init_formula):
         return OrderedDict()
 
     formula_dict = OrderedDict()
-    formula_dict = __parse_parentheses(init_formula, "1", formula_dict)
+    __parse_parentheses(init_formula, "1", formula_dict)
```

Comment:
I prefer that the parse parenthesis always returns a value, not a boolean. Could you please modify the change accordingly?

Reply:
Could you also add some unit tests for the methods that you have modified, please? Like: 
- __parse_parenthesis
- make_fraction_convertion in regex_parser.py as well.

Thank you